### ISSUE TITLE
Question on attribute predictor (classifiers' outputs and meaning of `attributes_5.json` dictionary)

### ISSUE THREAD

**Issue by chi0tzp** [yumingj/Talk-to-Edit#1]
Hi, 

I'm trying to use your pre-trained classifier for the five CelebA attributes that you use (Bangs, Eyeglasses, No_Beard, Smiling, Young). I'm building the model that you provide (the modified ResNet) using `attributes_5.json` and I load the weights given in `eval_predictor.pth.tar`.

As far as I can tell, for each of the above five attributes, you have a classification head. For instance, `classifier32Smiling` which, at the top of it, it has a linear layer with 6 outputs. This is determined by the sub-dictionary
```
"32": {
            "name": "Smiling",
            "value":[0, 1, 2, 3, 4, 5],
            "idx_scale": 1,
            "idx_bias": 0
        }
```
found in `attributes_5.json`. Similarly you build the rest of the classifiers. My question is why you use these `value` lists above (i.e., "value":[0, 1, 2, 3, 4, 5])? What are those classes?

I'd like to use this model for predicting a score for each of the five attributes for a batch of images. Would you think this is possible?

As a side note, the function that you use for post-processing the predictions, i.e., `output_to_label`, gives NaNs in many cases. This is due to high values in predictions (in my case), which lead exp(.) to get to Inf, and thus softmax be NaN. Just says, that you could shift the maximum prediction to be zero before calculating softmax.

Thank you!
